jobs/integration/conftest.py

```diff
@@ -137,6 +140,17 @@ async def _load(self):
         self.charm_channel = request.config.getoption("--charm-channel")
         self.vault_unseal_command = request.config.getoption("--vault-unseal-command")
 
+    def juju_base(self, series):
+        """Retrieve juju 3.1 base from series."""
+        if self.juju_version < (3, 1):
+            return f"--series={series}"
+        mapping = {
+            "bionic": "ubuntu@18.04",
+            "focal": "ubuntu@20.04",
+            "jammy": "ubuntu@22.04",
+        }
+        return f"--base={mapping[series]}"
```

Comment:
I think '--base' is the one we need. However, after reading the 'juju deploy' [page](https://juju.is/docs/olm/juju-deploy), it seems that '--base' does something else (Existing storage to attach to the deployed unit). We should watch out for any unusual issues, but it's likely just a documentation issue.